src/main/java/io/jenkins/pluginhealth/scoring/PluginHealthScoring.java

```diff
@@ -24,14 +24,30 @@
 
 package io.jenkins.pluginhealth.scoring;
 
+import java.io.IOException;
+
+import io.jenkins.pluginhealth.scoring.cli.ImportPluginCLR;
+
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication(scanBasePackages = "io.jenkins.pluginhealth.scoring")
 @EnableScheduling
-public class PluginHealthScoring {
+public class PluginHealthScoring implements CommandLineRunner {
```

Comment:
Definitively not that. We removed that because you have all your tests using the app that will run the code.